packages/themes/default/src/components/pagination.scss

```diff
@@ -6,49 +6,53 @@
 		font-family: var(--font-family);
 	}
 
-	.button:focus {
-		outline: none;
-	}
-
-	.button-inner {
-		background-color: var(--color-light);
-		border-radius: var(--border-radius);
-		border: rem(1) solid var(--color-primary);
-		color: var(--color-primary);
-		font-weight: 700;
-		min-height: var(--a11y-min-size);
-		min-width: var(--a11y-min-size);
-		padding: rem(8);
-		text-align: center;
-		transition-duration: 0.5s;
-		transition-property: background-color, color, border-color;
-	}
-
-	.button:focus .button-inner {
-		@include focus-outline;
-	}
-
-	.button:is(:active, :hover):not(:disabled) .button-inner {
-		background-color: var(--color-primary-variant);
-		border-color: var(--color-primary-variant);
-		box-shadow: 0 rem(2) rem(8) rem(2) rgba(8, 35, 48, 0.24);
-		color: var(--color-light);
-	}
-
-	.button:active .button-inner {
-		color: var(--color-light);
-		outline: none;
-	}
-
-	.selected {
-		button {
-			opacity: 1;
-		}
-
-		.button-inner {
-			background-color: var(--color-primary);
-			color: var(--color-light);
-			border: 0;
+	.kol-pagination {
+		&__navigation-list {
+			.button:focus {
+				outline: none;
+			}
+
+			.button-inner {
+				background-color: var(--color-light);
+				border-radius: var(--border-radius);
+				border: rem(1) solid var(--color-primary);
+				color: var(--color-primary);
+				font-weight: 700;
+				min-height: var(--a11y-min-size);
+				min-width: var(--a11y-min-size);
+				padding: rem(8);
+				text-align: center;
+				transition-duration: 0.5s;
+				transition-property: background-color, color, border-color;
+			}
+
+			.button:focus .button-inner {
+				@include focus-outline;
+			}
+
+			.button:is(:active, :hover):not(:disabled) .button-inner {
+				background-color: var(--color-primary-variant);
+				border-color: var(--color-primary-variant);
+				box-shadow: 0 rem(2) rem(8) rem(2) rgba(8, 35, 48, 0.24);
+				color: var(--color-light);
+			}
+
+			.button:active .button-inner {
+				color: var(--color-light);
+				outline: none;
+			}
+
+			.selected {
+				button {
+					opacity: 1;
+				}
+
+				.button-inner {
```

Comment:
Bitte soweit möglich immer nach Elementen gruppieren. Siehe auch: https://github.com/public-ui/kolibri/wiki/Code-Style-and-Conventions#bem

Dadurch, dass wir in dieser Komponente nicht zwischen mehreren Buttons unterscheiden müssen, ist es eigentlich nicht notwendig, die Button-Stylings in `__navigation-list` zu verschachteln.

Ich habe das hier einmal exemplarisch umgesetzt: https://gist.github.com/sdvg/0ab78974e3c7e7e6113862885cf67754